ratpack/src/main/groovy/com/baeldung/Ratpack.groovy

```diff
@@ -0,0 +1,79 @@
+package com.baeldung;
+
+@Grab('io.ratpack:ratpack-groovy:1.6.1')
+import static ratpack.groovy.Groovy.ratpack
+
+import com.baeldung.model.User
+import com.google.common.reflect.TypeToken
+import ratpack.exec.Promise
+import ratpack.handling.Context
+import ratpack.jackson.Jackson
+import groovy.sql.Sql
+import java.sql.Connection
+import java.sql.PreparedStatement
+import java.sql.ResultSet
+import ratpack.hikari.HikariModule
+import javax.sql.DataSource;
+
+ratpack {
+    serverConfig {
+        port(5050)
+    }
+    bindings {
+        module(HikariModule) { config ->
+            config.dataSourceClassName = 'org.h2.jdbcx.JdbcDataSource'
+            config.addDataSourceProperty('URL', "jdbc:h2:mem:devDB;INIT=RUNSCRIPT FROM 'classpath:/User.sql'")
+        }
+    }
+
+    handlers {
+
+        get { render 'Hello World from Ratpck with Groovy!!' }
+
+        get("greet/:name") { Context ctx ->
+            render "Hello " + ctx.getPathTokens().get("name") + "!!!"
+        }
+
+        get("data"){
+            render Jackson.json([title:"Mr",name:"Norman",country:"USA"])
```

Comment:
`MINOR` - the spacing is missing between commas

Reply:
Fixed